assets/js/modules/adsense/datastore/index.js

```diff
@@ -51,11 +57,17 @@ const baseModuleStore = Modules.createModuleStore( 'adsense', {
 	adminPage: 'googlesitekit-module-adsense',
 } );
 
+const existingTagStore = createExistingTagStore( 'modules', 'adsense', {
+	tagMatchers,
+	isValidTag: isValidClientID,
+} );
```

Comment:
I wonder whether we should handle this inside of the `tags.js` file? This is where the part of the store was defined before, and also `tags.test.js` tests the `getExistingTag` selector. So I think we should move this back into the `tags.js` file, it already has a `Data.combineStores` call anyway.

Reply:
@felixarntz oh yeah, good call. It makes much more sense calling the store creation in the `tags.js` files. I'll move it in there.